datahub_sap_hana/ingestion.py

```diff
@@ -147,15 +173,14 @@ def _get_view_lineage_elements(self) -> Dict[Tuple[str, str], List[str]]:
                     self.config.env,
                 )
             )
-
         return lineage_elements
 
     def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
         """Creates MetadataWorkUnit objects for table lineage based on the downstream and downstream objects from the query results.
         Returns an iterable MetadataWorkUnit that are emitted to Datahub.
         """
-
-        lineage_elements = self._get_view_lineage_elements()
+        conn = self.get_db_connection()
```

Comment:
I would actually do this one level higher, in `get_workunits` and then pass the `conn` here